apps/projects/contracts/Projects.sol

```diff
@@ -1,14 +1,66 @@
 pragma solidity ^0.4.24;
 
 import "@tps/test-helpers/contracts/apps/AragonApp.sol";
+import "@tps/test-helpers/contracts/lib/zeppelin/math/SafeMath.sol";
+
+/*******************************************************************************
+    Copyright 2018, That Planning Suite
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*******************************************************************************/
+/*******************************************************************************
+* @title Projects Contract
+* @author Kevin Siegler
+* @dev This contract defines a registry for Github issues in addition to
+* applying bulk bounties..
+*******************************************************************************/
+interface Bounties {
+    function issueBounty(
+        address _issuer,
+        uint _deadline,
+        string _data,
+        uint256 _fulfillmentAmount,
+        address _arbiter,
+        bool _paysTokens,
+        address _tokenContract
+    ) external returns (uint);  
+
+    function activateBounty(
+        uint _bountyId, 
+        uint _value
+    ) external payable;
+
+    function acceptFulfillment(
+        uint _bountyId, 
+        uint _fulfillmentId
+    ) external;
+}
 
 
 contract Projects is AragonApp {
-    function initialize( // solium-disable-line blank-lines
-        //Vault _vault
```

Comment:
@capsulecorplab Can we actually put all vault references back in? We will eventually incorporate the vault here. If you feel strongly against it, we don't have to include it, but I think it's good way to demonstrtate near-term intent to add it.

Reply:
SGTM

Reply:
@topocount We still have it uncommented in [L11](https://github.com/spacedecentral/planning-suite/pull/257/files/9ee12d3dcb75ebb8548d970535b1abfd33f2b8ac#diff-8bb0f56b9a5eaab6509b2ba5a2518e0cL11)